### PR TITLE
fix(detail): fix cosmetic issues in detail and quick-preview

### PR DESCRIPTION
**Pull request by SMahil**
OSIO Issue: https://openshift.io/openshiftio/openshiftio/plan/detail/1913

### CODE REVIEW

**Comment by fabric8cd** [comment]
@SMahil fabric8/fabric8-ui:SNAPSHOT-PR-2477-1 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2477-fabric8-planner.badger.fabric8.io

**Comment by nimishamukherjee** [comment]
Need to verify on ngrx version

**Comment by sanbornsen** [comment]
@SMahil the fixes need to be done on the same files in **componets_ngrx**

**Comment by fabric8cd** [comment]
@SMahil fabric8/fabric8-ui:SNAPSHOT-PR-2477-3 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2477-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@SMahil fabric8/fabric8-ui:SNAPSHOT-PR-2477-5 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2477-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@SMahil fabric8/fabric8-ui:SNAPSHOT-PR-2477-6 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2477-fabric8-planner.badger.fabric8.io

**Comment by Raunak1203** [comment]
@SMahil please see the dropdown in the screenshot
screenshot from 2018-03-21 16-17-02

File: src/app/components_ngrx/work-item-quick-preview/work-item-quick-preview.component.ts [hunk 1]
```diff
@@ -0,0 +1,418 @@
...
+    this.dropdownRef.openDropdown();
+  }
+
+  closeDropdown() {
+    this.dropdownRef.closeDropdown();
+  }
+  closeAddAssignee() {
+    this.activeAddAssignee = false;
+  }
+}
```
Comment (vikram-raj) [inline src/app/components_ngrx/work-item-quick-preview/work-item-quick-preview.component.ts]: Delete this file. This file is already deleted in master. we are not using this.

File: src/app/components/work-item-detail/work-item-detail.component.html [hunk 2]
```diff
@@ -274,6 +364,19 @@ <h3 *ngIf="workItem.id" class="detail-id" id="wi-detail-id">{{workItem.attribute
...
+      <!-- <alm-work-item-link
+        [workItem]="workItem"
+        [loggedIn]="loggedIn">
+      </alm-work-item-link> -->
+      <work-item-comment-wrapper
+        [workItem]="workItem"
+        [loggedIn]="loggedIn"
+        [loggedInUser]="loggedInUser">
+      </work-item-comment-wrapper>
+>>>>>>> fix(ngrx):fix detail and quick preview css in ngrx:src/app/components_ngrx/work-item-quick-preview/work-item-quick-preview.component.html
```
Comment (vikram-raj) [inline src/app/components/work-item-detail/work-item-detail.component.html]: Resolve the conflicts. Otherwise, the build will never pass.

**Comment by rgarg1** [comment]
LGTM @SMahil . _Add label_ was recently modified by @debloper to _Assign Label(s)_. I hope this PR doesn't overwrite that change.

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @SMahil for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


For more information please head over to official [documentation](http://arquillian.org/ike-prow-plugins/#_test_keeper_plugin). You can find there how to configure the plugin - for example exclude certain file types so if PR contains only them it won't be checked.

**Comment by alien-ike** [comment]
Hey @SMahil! It seems you tried to trigger `/ok-without-tests` command, but this will not have any effect due to insufficient permission. You have to be admin or requested reviewer, but not pull request creator for this command to take an effect.

**Pull request merged**